src/Copy/Text.elm

```diff
@@ -432,52 +432,43 @@ t key =
 
         -- Get Help page
         GetHelpTitle ->
-            "[cCc] People You Can Talk To"
+            "People you can talk to"
 
         GetHelpSection1Title ->
-            "[cCc] Support from Survivors"
+            "Join the SEA survivors’ forum"
 
         GetHelpSection1Quote1 ->
-            ""
-
-        GetHelpSection1Quote2 ->
-            ""
+            "“Meeting other people who’ve been through what I’ve been through is such a massive help. It’s amazing to know people who can really understand what I went through.”"
 
         GetHelpSection1Description ->
-            ""
+            "To share your experience with others in our supportive community of economic abuse survivors, sign up to SEA’s Survivor Forum."
 
         GetHelpSection1Link ->
-            "[cCc] SEA Survivor Forum"
+            "[cCc] Join SEA Survivor Forum"
 
         GetHelpSection2Title ->
-            "[cCc] Specialist Debt Advice"
+            "The Financial Support Line for Victims of Domestic Abuse"
 
         GetHelpSection2Quote1 ->
-            ""
-
-        GetHelpSection2Quote2 ->
-            ""
+            "“When I found this number, I was a huge relief. It made all the difference knowing I could get the right advice for what I was going through by someone who understand.”"
```

Comment:
I've left in the handed quotes from the copy here. As we have a lot of quotes in the site it would be good to make a common strategy eg handed quotes in copy / escaped normal quotes / CSS pseudo content adds them

Reply:
I think I prefer the blockquote or q html tag + css

Reply:
I agree, I'll change this now - no quotes allowed in copy. I think `<q>` is for inline quotes only